addons/binding/org.openhab.binding.heos/README.md

```diff
@@ -0,0 +1,427 @@
+For further information about notifications also refer to the [OpenHab documentation](http://docs.openhab.org/configuration/multimedia.html).
+
+The URL defines the base address where the sound files are located. For example if the files are placed within the OpenHab directory \openHAB-conf\sounds the callback address has be set to the OpenHab server address inclusive the port.  
+
+Example: http://192.168.0.7:8080 if the sound file can be accessed there, or if the sound file is located on the OpenHab server http://localhost:8080 for example.
+
+
+## Thing Configuration
+
+**It is recommended to configure the things via the PaperUI or HABmin**
+
+### Bridge Configuration
+
+The bridge can be added via the PaperUI. After adding the bridge the user name and password can be set by editing the thing via the PaperUI. For manual configuration the following parameters can be defined. The ipAddress has to be defined. All other fields are optional.
+
+````
+Bridge heos:bridge:main "name" [ipAddress="192.168.0.1", name="Default", unserName="xxx", password="123456"]  
+````
+
+### Player Configuration
+
+Players can be added via the PaperUI. All fields are then filled automatically.
+For manual configuration a player is defined as followed:
+
+````
+Thing heos:player:pid "name" [pid="123456789", name="name", model="modelName", ipAddress="192.168.0.xxx", type="Player"] 
+````
+
+PID behind the heos:player:--- should be changed as required. Every name or value can be used. It is recommended to use the player PID. Within the configuration the PID as well as the type field is mandatory. The rest is not required.
+If the PID isn't known it can be discovered by establishing a TelNet connection (port 1255) to one player and search for available players
+(Command: heos://player/get_players)
+within the network. For further details refer to the [HEOS CLI](http://rn.dmglobal.com/euheos/HEOS_CLI_ProtocolSpecification.pdf) specification.
+
+
+*If player is configured by PaperUI the UID of the player is equal to the player ID (PID) from the HEOS system*
+
+### Group Configuration
+
+```
+Thing heos:group:memberHash "name" [gid="123456789", name="name", model="modelName", ipAddress="192.168.0.xxx", type="Group", memberHash="123456789"] 
+```
+
+*If group is configured by PaperUI the group UID is calculated by the Hash value of the group members*
+
+Required fields are the GID which is the PID of the group leading player and the type which is Group here.  
+
+### Defining Bridge and Players together
+
+Defining Player and Bridge together. To ensure that the players and groups are attached to the bridge the definition can be like:
+
+```
+Bridge heos:bridge:main "Bridge" [ipAddress="192.168.0.1", name="Bridge", userName="userName", password="123456"] {
```

Comment:
How why / when do I need to enter a password? Its not directly clear from the README and the GUI for me.

Reply:
I added an description which password it is below the Bridge Configuration Chapter